app/sampleUsers.js

```diff
@@ -0,0 +1,43 @@
+module.exports = [
+    {
+      firstName: 'Sonali',
+      lastName: 'Kumari',
+      email: 'sonali211103@gmail.com',
+      password: 'password1',
+      dob: '21-11-2003',
+      mobileNo: '1234567890'
+    },
+    {
+      firstName: 'Jane',
+      lastName: 'Doe',
+      email: 'jane@example.com',
+      password: 'password2',
+      dob: '05-05-1995',
+      mobileNo: '9876533210'
+    },
+    {
+        firstName: 'John',
+        lastName: 'Doe',
+        email: 'john@example.com',
+        password: 'password3',
+        dob: '08-05-1998',
+        mobileNo: '9876553210'
+      },
+      {
+        firstName: 'Aayush',
+        lastName: 'Sharma',
+        email: 'aayush@example.com',
+        password: 'password4',
+        dob: '06-06-1996',
+        mobileNo: '9876563210'
+      },
+      {
+        firstName: 'Anmol',
+        lastName: 'Kumar',
+        email: 'anmol@example.com',
+        password: 'password5',
+        dob: '01-03-2001',
+        mobileNo: '9876543210'
+      },
+  ];
+  
```

Comment:
Please use .gitignore to not push node_modules these will make code base unnecessarily large